modules/nextflow/src/main/groovy/nextflow/util/ThreadPoolBuilder.groovy

```diff
@@ -160,17 +161,14 @@ class ThreadPoolBuilder {
         return result
     }
 
-
     static ThreadPoolExecutor io(String name=null) {
         io(10, 100, 10_000, name)
     }
 
-
     static ThreadPoolExecutor io(int min, int max, int queue, String name=null) {
         new ThreadPoolBuilder()
                 .withMinSize(min)
                 .withMaxSize(max)
-                .withQueueSize(queue)
```

Comment:
Why is this removed?

Reply:
The idea was to default on unbound queue, but I agree that's inconsistent. Above all it's not used anymore, it should be removed

Reply:
Removed